app/code/Magento/Sitemap/Model/ResourceModel/Catalog/Product.php

```diff
@@ -114,6 +124,7 @@ public function __construct(
         \Magento\Catalog\Model\ResourceModel\Product\Gallery $mediaGalleryResourceModel,
         \Magento\Catalog\Model\Product\Gallery\ReadHandler $mediaGalleryReadHandler,
         \Magento\Catalog\Model\Product\Media\Config $mediaConfig,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
```

Comment:
Please, inject new dependencies according to our [Backward compatible development policy](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter)